packages/firebase_storage/firebase_storage/example/test_driver/test_utils.dart

```diff
@@ -14,6 +14,15 @@ Random _random = Random();
 String _getRandomString(int length) => String.fromCharCodes(Iterable.generate(
     length, (_) => _chars.codeUnitAt(_random.nextInt(_chars.length))));
 
+String get testEmulatorHost {
+  if (defaultTargetPlatform == TargetPlatform.android && !kIsWeb) {
+    return '10.0.2.2';
+  }
+  return 'localhost';
+}
+
+const int testEmulatorPort = 9080;
```

Comment:
use standard / default storage emulator port

```suggestion
const int testEmulatorPort = 9199;
```